### PR TITLE
revert copy of generated S files to old copy

### PR DESCRIPTION
**Pull request by strichmo**
the rsync did not seem to work with some users

This change will effectively revert this PR: https://github.com/openhwgroup/core-v-verif/pull/826

On some systems, the rsync simply did not work properly.

### CODE REVIEW

**Review by MikeOpenHWGroup — APPROVED** [review]
This PR will need a Committer to approve it (either me or @strichmo) so I am doing so now.  However, I will not merge until @silabs-hfegran has also reviewed and approved it.

**Comment by silabs-hfegran** [comment]
The old code works, so I don't have any objections to reverting back to that. 

On that note, is the complexity of the rm and copy portions really necessary, or could we perhaps make do with a simple cp *.S and rm *.S? Perhaps I'm overlooking something...

**Review by silabs-hfegran — APPROVED** [review]

**Review by silabs-hfegran — APPROVED** [review]

**Pull request merged**
